SplunkAppForWazuh/appserver/static/js/controllers/agents/cve/agents-cve.html

```diff
@@ -0,0 +1,100 @@
+  <div layout="row" class="wz-height-35">
+    <div class='wz-breadcrumb-margin'>
+      <a class="wz-text-link cursor-pointer" ui-sref='agents'>Agents</a>
+      <span> / </span>
+      <span ng-if="!agent.error" ui-sref='agent-overview({id:agent.id})'
+        class="wz-text-link cursor-pointer">{{agent.name}} ({{agent.id}})</span>
+      <span ng-if="agent.error">Unknown agent</span>
+      <span> / </span>
+      <span>Vulnerabilities</span>
+      <span ng-if="agent.status">
+        <span class="no-wrap wz-padding-left-8" aria-hidden="false">
+          <wz-svg icon="agent-{{agent.status}}"></wz-svg>
+        </span>{{formatAgentStatus(agent.status)}}
+      </span>
+    </div>
+    <div flex></div>
+    <!-- Report button -->
+    <div style="display:flex; padding-right: 10px;">
+      <button ng-if="reportingEnabled" md-no-ink class="btn wz-button-empty wz-button-report"
+        ng-disabled="loadingVizz || loadingReporting" ng-click="startVis2Png()" aria-label="Generate report button">
+        <i class="fa fa-fw fa-print" aria-hidden="true"></i>
+        <md-tooltip md-direction="bottom" class="wz-tooltip">
+          Generate report
+        </md-tooltip>
+      </button>
+      <wz-discover
+        breadcrumbs="{section: 'Agents', subSection: 'Vulnerabilities', agentName: agent.name, agentId: agent.id, ref: 'agents', agentRef: 'agent-overview'}">
+      </wz-discover>
+    </div>
+    <div style="margin-right:7px;" id='timePicker'></div>
+  </div>
+
+  <!-- Loading report-->
+  <div align='center' ng-if='loadingReporting'>{{reportStatus}}<br><i class="fa fa-fw fa-spin fa-spinner"
+      aria-hidden="true"></i></div>
+
+  <!-- Threat Detection and Response navbar -->
+  <md-nav-bar class="wz-nav-bar wz-margin-10 no-margin-left" md-selected-nav-item="'inventory'">
+    <md-nav-item class="wz-nav-item" md-nav-click="stopPropagation()" name="inventory">Inventory</md-nav-item>
+    <md-nav-item class="wz-nav-item" md-nav-click="stopPropagation()" 
+      ui-sref="ag-vul" name="vulnerabilities">Vulnerabilities</md-nav-item>
+    <md-nav-item ng-if="extensions.osquery === 'true'" class="wz-nav-item" md-nav-click="stopPropagation()"
+      ui-sref="ag-osquery" name="osquery">Osquery</md-nav-item>
+    <md-nav-item ng-if="extensions.virustotal === 'true'" class="wz-nav-item" md-nav-click="stopPropagation()"
+      ui-sref="ag-virustotal" name="virustotal">VirusTotal</md-nav-item>
+    <md-nav-item ng-if="extensions.docker === 'true'" class="wz-nav-item" md-nav-click="stopPropagation()"
+      ui-sref="ag-docker" name="docker">Docker listener</md-nav-item>
+  </md-nav-bar>
+
+
+  <div>
+    <!-- Filter bar -->
+    <wazuh-bar></wazuh-bar>
+
+    <div ng-show="loadingVizz">
+      <div align='center'>
+        Fetching data...<br></b><i class="fa fa-fw fa-spin fa-spinner" aria-hidden="true"></i>
+      </div>
+    </div>
+    
+    <div layout="row" class="wz-padding-top-0" ng-if="agent && agent.os && agent.os.platform !== 'windows'">
```

Comment:
Shows only on Linux systems. If Windows systems use the exact same view `agent.os.platform != windows` must be removed